File: golang/lab1.go
package main

import (
	"math"
)

func function(a, x float64) float64 {
	var y float64 = math.Pow(a, (math.Pow(x, 2)-1)) - math.Log10(math.Pow(x, 2)-1) + math.Pow((math.Pow(x, 2)-1), (1.0/3.0))
	return y
}

func zadacha1(a float64) []float64 {
	var zadacha []float64
	for x := 1.2; x < 4.2; x = x + 0.6 {
		zadacha = append(zadacha, function(a, x))
	}
	return zadacha
}

func zadacha2(a float64) []float64 {
	var zadachaB []float64
	var znach = [5]float64{1.28, 1.36, 2.47, 3.68, 4.56}
	for _, x1 := range znach {
		zadachaB = append(zadachaB, function(a, x1))
	}
	return zadachaB
}

File: lab1.py
import math
a = 2


def y(x):
    y = ((math.log10(a + x)) ** 2) / (a + x) ** 2
    return y


x = 1.2
while x < 4.2:
    print('Task A', y(x))
    x += 0.6


xl = [1.16, 1.32, 1.47, 1.65, 1.93]
lenx = len(xl)
for i in range(0, lenx):
    x = xl[i]
    print('Task B', y(x))
